src/app/saga/router.saga.ts

```diff
@@ -0,0 +1,18 @@
+import { all, takeLatest, put } from 'redux-saga/effects';
+import * as initActions from '../actions/init.actions';
+import { LOCATION_CHANGE, LocationChangeAction } from 'connected-react-router';
+
+const articleIdRegex = /A?articleId=[^&]+&*/;
```

Comment:
`A?articleId=([^&]+)&*` with this regex you should then be able to just do `found[1]`